pkg/logs/tailers/file/tailer.go

```diff
@@ -197,6 +201,14 @@ func addToTailerInfo(k, m string, tailerInfo *status.InfoRegistry) {
 	tailerInfo.Register(newInfo)
 }
 
+func getFileReadBufferSizeFromConfig() int {
```

Comment:
NIT: perhaps this should live in [config_keys.go](https://github.com/DataDog/datadog-agent/blob/977ffd14b0c5c9bc4bee995dd95bed0a31a3b55f/comp/logs/agent/config/config_keys.go#L111)?
We could also define the `1024 * 4` default value as a constant somewhere such that we don't have to keep it up to date here and in `setup/config.go`

Reply:
It looks like the `LogsConfigKeys` instance is only a thing while configuring endpoints? I could consider instantiating one with `NewLogsConfigKeys` if you think it's the best approach, I'm not very familiar with this I'll do what you think is best.
I'll create a const 👍